### PR TITLE
Aws bucket

### PR DESCRIPTION
**Pull request by shermanyin**

### CODE REVIEW

File: modules/aws/cac/main.tf [hunk 1]
```diff
@@ -10,20 +10,34 @@ locals {
...
+      cam_url                  = var.cam_url,
+      cac_installer_url        = var.cac_installer_url,
+      cac_token                = var.cac_token,
+      pcoip_registration_code  = var.pcoip_registration_code,
+
+      domain_controller_ip     = var.domain_controller_ip,
+      domain_name              = var.domain_name,
+      domain_group             = var.domain_group,
+      service_account_username = var.service_account_username,
+      service_account_password = var.service_account_password,
```
Comment (Edwin-Pau) [inline modules/aws/cac/main.tf:30]: Should we make the variable names the same between gcp and aws? I can do a separate commit to change to ad_service_account_username and ad_service_account_password

File: deployments/aws/single-connector/main.tf [hunk 2]
```diff
@@ -7,6 +7,22 @@
...
   prefix = var.prefix != "" ? "${var.prefix}-" : ""
+  bucket_name = "${local.prefix}pcoip-scripts-${random_id.bucket-name.hex}"
+}
+
+resource "random_id" "bucket-name" {
+  byte_length = 3
+}
+
+resource "aws_s3_bucket" "scripts" {
+  bucket        = local.bucket_name
```
Comment (Edwin-Pau) [inline deployments/aws/single-connector/main.tf:18]: In the GCP resource, this is `name          = local.bucket_name`

File: deployments/aws/single-connector/main.tf [hunk 3]
```diff
@@ -7,6 +7,22 @@
...
+  region        = var.aws_region
+  acl           = "private"
+  force_destroy = true
+
+  tags = {
+    Name = local.bucket_name
+  }
```
Comment (Edwin-Pau) [inline deployments/aws/single-connector/main.tf:25]: This block is not present in the GCP bucket resource.

**Review by Edwin-Pau — COMMENTED** [review]
Finished going through everything, looks good!

File: modules/aws/cac/main.tf (hunk 1 above)
Comment (shermanyin) [reply modules/aws/cac/main.tf:30]: Could you create a new patch to update those variable names like in GCP please?

File: deployments/aws/single-connector/main.tf (hunk 2 above)
Comment (shermanyin) [reply deployments/aws/single-connector/main.tf:18]: GCP and AWS are different resources and have different properties

File: deployments/aws/single-connector/main.tf (hunk 3 above)
Comment (shermanyin) [reply deployments/aws/single-connector/main.tf:25]: GCP and AWS are different resources and have different properties

**Pull request merged**